protos/bottle/inventory/v1.proto

```diff
@@ -6,4 +6,5 @@ package bottle.inventory;
 
 service V1 {
   rpc ComponentAvailabilityList(bottle.inventory.v1.ComponentAvailabilityListRequest) returns (stream bottle.inventory.v1.ComponentAvailabilityListResponse);
+  rpc InventoryLocations(bottle.inventory.v1.InventoryLocationsRequest) returns (stream bottle.inventory.v1.InventoryLocationsResponse); 
```

Comment:
What is the point to this endpoint? It looks like it takes a list and returns a single one...

Reply:
It takes a list of location ids and streams back the locations. We'll need this for the UI.

Reply:
I'd change the name of it to match the one above then. Also remove `Inventory` because it's already in the inventory context.

```suggestion
  rpc LocationList(bottle.inventory.v1.LocationListRequest) returns (stream bottle.inventory.v1.LocationListResponse); 
```